connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoConstants.scala

```diff
@@ -25,4 +25,8 @@ object KustoConstants {
   val DefaultBatchingLimit: Int = 100
   val DefaultExtentsCountForSplitMergePerNode: Int = 400
   val DefaultMaxRetriesOnMoveExtents: Int = 10
+  val DefaultExecutionQueueing: Int = TimeUnit.SECONDS.toMillis(15).toInt
```

Comment:
Is this 15msec or 15000msec?
(Sorry, not my native language)

Reply:
15000msec